contrib/linux-kernel/lib/zstd/compress.c

```diff
@@ -587,8 +587,10 @@ ZSTD_STATIC size_t ZSTD_compressSequences(ZSTD_CCtx *zc, void *dst, size_t dstCa
 {
 	const int longOffsets = zc->params.cParams.windowLog > STREAM_ACCUMULATOR_MIN;
 	const seqStore_t *seqStorePtr = &(zc->seqStore);
-	U32 count[MaxSeq + 1];
-	S16 norm[MaxSeq + 1];
+	U32 *count = (U32 *)zc->tmpCounters;
+	S16 *norm = (S16 *)(count + MaxSeq + 1);
```

Comment:
Since the next usage is inside of `FSE_.*()` functions, to make that work here I would have to put `norm` at the end of the workspace. Do you think its worth it?

I'll definitely update the other places though.